book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    @rentals = Rental.new(date, person, self)
```

Comment:
- [ ] **Required:** Kindly observe that you've mixed up the order in which you provided the input to the `Rental` instance different from the way you defined them in the class. Kindly fix the arrangement to avoid errors, it should be the date, self (which is the book) and person.

https://github.com/Carshy/oop-school-library/blob/ab1b3730339c9929c72e239503f1788d400d4c8d/rental.rb#L2